docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,257 @@
+# Earth Observation guide
+
+This guide aims to help users to work with Earth Observation (EO) data using CSC's computing resources. The purpose of this guide is to help you find the right data and tools for your raster data based EO tasks. If you are interested in the fundamentals of EO, please check the resources at the bottom of the page.
+
+**What are the benefits of using EO data**
+
+* Possibility to observe wide area at same time
+* Non-intrusive
+* Same sensor for different parts of the world
+* Time series 
+
+!!! default "Raster data format"
```

Comment:
I find this raster explanation to be rather weak, so still suggest replacing with a link to longer explanation.